packages/itwinui-css/src/carousel/carousel.scss

```diff
@@ -68,10 +68,8 @@
     }
 
     &::after {
+      @include iui-icon-style('s');
       content: '';
-      display: block;
```

Comment:
Why was `display: block` removed if `iui-icon-style` has `display: flex`? I saw the same thing in `radio-tile.scss` as well

Reply:
I was testing to see if `@include iui-icon-style('s')` should go before or after `display: block` to see if the display type mattered, and it worked with `display: flex` so instead of putting the `include` after `display: block` I just removed it.